packages/grid_client/src/modules/k8s.ts

```diff
@@ -143,22 +143,25 @@ class K8sModule extends BaseModule {
         throw new ValidationError(`Another master with the same name ${master.name} already exists.`);
       masters_names.push(master.name);
 
-      if (network) break;
       // Sets the network and contractMetadata based on the node's zos version
       const nodeTwinId = await this.capacity.getNodeTwinId(master.node_id);
       const features = await this.rmb.request([nodeTwinId], "zos.system.node_features_get", "", 20, 3);
       if (features.some(item => item.includes("zmachine-light") || item.includes("network-light"))) {
-        network = new ZNetworkLight(options.network.name, options.network.ip_range, this.config);
-        await network.load();
+        if (!network) {
```

Comment:
Will this network be defined at any point?

Reply:
please check that in the rest of the changes

Reply:
> Will this network be defined at any point?

Yes it will be defined in the next iteration also in the workers